create/create_test.go

```diff
@@ -95,7 +97,7 @@ func TestMergeTemplatesJSON(t *testing.T) {
 		Development: core.Development{
 			Template: "typescript-react",
 			RootDir:  rootDir,
-			Renderer: core.Renderer{Name: "@shopify/admin-ui-extension"},
+			Renderer: core.Renderer{Name: "@shopify/admin-ui-extension", Version: "latest"},
```

Comment:
Should we update to use a real version to make the test match more closely with the real expected value?